inventory/s3/orc_reader.go

```diff
@@ -57,7 +57,7 @@ func (r *OrcInventoryFileReader) inventoryObjectFromRow(rowData []interface{}) I
 	}
 	var lastModified *int64
 	if lastModifiedIdx, ok := r.orcSelect.IndexInSelect["last_modified_date"]; ok && rowData[lastModifiedIdx] != nil {
-		lastModified = swag.Int64(rowData[lastModifiedIdx].(time.Time).Unix())
+		lastModified = swag.Int64(rowData[lastModifiedIdx].(time.Time).UnixNano() / int64(time.Millisecond))
```

Comment:
Please use  [`time.Unix`](https://golang.org/pkg/time/#Unix) rather than perform this conversion.  It would be less typing, also because my reading of the documentation]says it doesn't work:
>  The zero value of type Time is January 1, year 1, 00:00:00.000000000 UTC. As this time is unlikely to come up in practice, the IsZero method gives a simple way of detecting a time that has not been initialized explicitly. 
Also I don't understand how you can cast an integer type to `time.Time`, which is [a big-ish struct](https://golang.org/src/time/time.go?s=6278:7279#L117).

Does this function run in any test?

Reply:
After f2fing I realize I'm holding it backwards.  Just rename to `lastModifiedMillis` and it will be clearer.